sdks/python/apache_beam/runners/interactive/interactive_runner.py

```diff
@@ -131,7 +132,7 @@ def end_session(self):
     if exit is not None:
       self._in_session = False
       _LOGGER.info('Ending session.')
-      exit(None, None, None)
+      sys.exit(None, None, None)
```

Comment:
Can you also revert this change? We're looking for the exit in line 131 provided by the underlying runner. I think that's causing the precommit test errors.

Reply:
Done. I've added the `consider-using- sys-ext` to the ignored pylint list